Assets/7-Babybaras/Scripts/Lassoscript.cs

```diff
@@ -0,0 +1,95 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+namespace Babybaras
+{
+    public class Bugvars
+    {
+        public static bool gotWaterbug = false;
+        public static bool gotFirebug = false;
+        public static bool gotLightningbug = false;
+        public static int throwing = 1; // throwing is 1 is throwing, -1 if pulling
+        public static bool lassoing = false;
+        public static AudioSource bugSounds = Managers.AudioManager.CreateAudioSource();
```

Comment:
Would this still work if execution starts from a main menu, before starting up the minigames? This is a static field, which means that at time time this evaluates, Managers.AudioManager might be unset. Are you able to ensure that this works even if that is the case?